### DISCUSSION TITLE
interactive-design/streamdiffusion/stable-diffusion-texture-generate/

### DISCUSSION THREAD

**Discussion by giscus** [tedliou/blog-discussion#4] [General]
# unity/stable-diffusion-texture-generate/

想要設計好看的場景，但找不到貼圖素材嗎？Stable Diffusion 是一個免費的 AI 圖片生成工具，我們能用它來快速生成遊戲場景的貼圖，短時間內產出擁有高品質材質的物件。本文將教你如何透過簡單的設定來生成無縫貼圖，真正落實 AI 輔助設計，大幅提升開發效率！

https://tedliou.com/unity/stable-diffusion-texture-generate/

**Comment by tedliou** [comment]
這邊提供文章中輸入給 AI 的彩色貼圖和 Depth Map 根據輸入生出的高度和法線貼圖，可以先下載匯入到 Unity 中測試看看效果！

![](https://imgur.com/CTPcvFj.png)
![](https://imgur.com/xB7c8Oq.png)
![](https://imgur.com/QqMln5M.png)
